test/dlc_tests/eks/eks_manifest_templates/kubeflow/install_kubeflow_custom_kfctl.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
```

Comment:
Created a duplicate script as the original script (install_kubeflow.sh) will be used by cx for installing kubeflow.